src/lib/cli.ts

```diff
@@ -61,6 +62,53 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+type Update = { // eslint-disable-line no-unused-vars
+    latest: string,
+    current: string,
+    type: string,
+    name: string
+};
+
+const confirmLaunchInit = () => {
+    debug(`Initiating launch init confirm.`);
+
+    const question: Array<object> = [{
+        message: `Configuration for running sonar has not been set yet. Do you want to create a configuration file?`,
+        name: 'confirm',
+        type: 'confirm'
+    }];
+
+    return inquirer.prompt(question);
+};
+
+const loadOrCreateIfNotExits = async () => {
+    let config: IConfig;
+    let configPath: string;
+
+    try {
+        configPath = Config.getFilenameForDirectory(process.cwd());
+        debug(`Loading configuration file from ${configPath}.`);
+        config = Config.load(configPath);
+    } catch (err) {
+        if (err.message !== `Couldn't find any valid configuration`) {
```

Comment:
I don't like the idea to compare the `err.message`, if some day someone change the message, this code is not going to work as expected. However, I think we should offer them the option to create a new config file whatever the problem was. So this `if` is not necessary. (if so, maybe you need to change the `inquirer` message